app/controllers/new_user/dossiers_controller.rb

```diff
@@ -69,14 +70,14 @@ def update
       elsif draft?
         flash.now.notice = 'Votre brouillon a bien été sauvegardé.'
         render :modifier
+      elsif dossier.brouillon?
```

Comment:
Non bloquant : le maniaque en moi se demande si la modification de cette méthode ne devrait pas être faite dans un commit autre que `[Fix #1677] Prevent invites from submitting a dossier`, nommé par exemple `[Fix #1677] Allow invites to edit a dossier`

Reply:
Allez, je te l’ai fait :-)